### DISCUSSION TITLE
PVE - WORLD BOSS

### DISCUSSION THREAD

**Discussion by Procion06** [Crownicles/Crownicles#2584] [Nouvelles idées]
pop d'un World Boss une fois tous les 3 mois n'importe où sur la map !
il se déplace comme un joueur durant 48h et abîme les routes sur son passage avant de disparaître
le temps de trajet sur lequel il se trouve est alors doublé pour les joueurs à cause des dégâts qu'il fait en se déplaçant
il n'y a aucune restriction pour l'attaquer pendant ces 48H tant que les joueurs sont sur le même trajet (ou pas au choix)

le Boss a 1M de pv et s'il n'est pas tué durant ces 48h, il disparaît tout simplement
(ça correspond à 500 joueurs qui l'attaquent en faisant une moyenne de 2000 dégâts avant de mourir)
si le World Boss meurt il lâche un coffre sur le chemin où il a été tué
seuls les joueurs qui l'ont attaqué peuvent ouvrir ce coffre en se rendant sur ce chemin
le coffre dépope au bout de 24h

contenu du coffre :
de façon random, un des 5 artefacts du set d'Adamantium

- épée en adamantium
- bouclier en adamantium
- bague en adamantium (objet attaque)
- collier en adamantium (objet défense)
- cape en adamantium (objet vitesse)

**Comment by Ravenclaw-l** [comment]
Super idée ! Possible d'utiliser l'attaque guilde si plusieurs membres d'une guilde suivent le même itinéraire ?

**Reply by Procion06** [reply]
oui comme pour l'île, tous les membres d'une guilde sur le trajet du Boss ont l'attaque de guilde
